client/packages/invoices/src/InboundShipment/api/hooks/utils/useSelectedStockLineIds.ts

```diff
@@ -1,4 +1,4 @@
-import { useTableStore } from 'packages/common/src';
+import { useTableStore } from '@openmsupply-client/common';
```

Comment:
Ah sorry for this noise, cleaned up these imports (the 'packages/..' imports break the tests)